packages/ra-ui-materialui/src/list/FilterFormInput.js

```diff
@@ -43,7 +43,6 @@ const FilterFormInput = ({
                 </IconButton>
             )}
             {React.cloneElement(filterElement, {
-                allowEmpty: true,
```

Comment:
Not good: this is breaking existing forms. You should inspect if the child has defined allowempty first.

Reply:
:+1: